robottelo/hosts.py

```diff
@@ -1546,6 +1546,19 @@ def set_rex_script_mode_provider(self, mode='ssh'):
         if result.status != 0:
             raise SatelliteHostError(f'Failed to enable pull provider: {result.stdout}')
 
+    def run_installer_arg(self, *args, timeout='20m'):
```

Comment:
I think this should be moved in capsule midins somewhere rathan than keeping everything in hosts module.

Reply:
noted, but I'd do it separately as there are other capsule-specific functions in this file that could be moved as well but don't relate to this PR's main intent

Reply:
Sure! Please you can move all of them in mixins in this PR itself, or do later but please dont forget :)